### ISSUE TITLE
PS 23.5 Welcome screen blank

### ISSUE THREAD

**Issue by b9aa** [NGLEmp/NGLEmp#23]
PS 23.5 (com.adobe.ccx.start.5.11.0)
Welcome screen blank . (push Esc for exit)

NGLEmp 0.8.2.1 Until PS 23.4.2 (com.adobe.ccx.start.5.10.0)
Welcome screen work.

**Comment by upiter** [comment]
Does welcome screen work without NGLEmp installed in `23.5` in your case?
NGLEmp shouldn't affect welcome screen.

**Comment by b9aa** [comment]
If without NGLEmp installed ,welcome screen (5.11.) work ,and show can not verify account windows (both no login)
I found it if replace app.js file of com.adobe.ccx.start (5.11) from  com.adobe.ccx.start(5.10.0)
welcome screen work again ,so I guess maybe something change in 5.11 app.js for verify license?

**Comment by b9aa** [comment]
more information .
If use patch PS 23.5 by Adobe-GenP-2.7 only , Welcome screen work.(com.adobe.ccx.start.5.11)
but If use the patch and NGLEmp 0.8.2.1 ,Welcome screen blank (com.adobe.ccx.start.5.11).

maybe this information can help ?

**Comment by upiter** [comment]
@b9aa thanks you for additional details and comparison.

NGLEmp doesn't affect Welcome screen (no patches, no modifications).

If you could provide these files, it would be great:

- Original `com.adobe.ccx.start` version `5.10.0` (expected to be working solution)
- Original `com.adobe.ccx.start` version `5.11.0` (expected to be non-working solution)
- Modified `com.adobe.ccx.start` version `5.11.0` (expected to be working solution)

**Comment by b9aa** [comment]
Yes,I Know,but there was something change (license check or other)  in 5.11 app.js
because welcome screen always work fine with NGLEmp until 5.11

thank you for check .
Modified way is just copy the \js\app.js from 5.10 to replace \js\app.js in 5.11  

[both.zip](https://github.com/NGLEmp/NGLEmp/files/9413066/both.zip)
